README.md

```diff
@@ -102,6 +102,8 @@ Use "harbor [command] --help" for more information about a command.
 
 #### Config Management
 
+After you successfully login with your credentials using the `harbor login` command, the details are stored as a config file. 
```

Comment:
where?
Maybe it would help indicating that to the user where configs are stored by default